monarch/monarch.lua

```diff
@@ -509,7 +509,9 @@ local function load(screen)
 		msg.post(screen.proxy, MSG_ENABLE)
 	elseif screen.factory then
 		screen.factory_ids = collectionfactory.create(screen.factory)
-		screen.transition_url = screen.factory_ids[screen.transition_id]
+		if screen.transition_id then
```

Comment:
Thanks for opening this issue.

Wouldn't this cause the `screen.transition_url` to be nil?
Since the string `__DEPRECATED__` is set by default, I thought it would be better to use 
`if not screen.transition_id == "__DEPRECATED__"` instead.


<img width="410" alt="image" src="https://github.com/britzl/monarch/assets/46726330/bedb38a6-f9f5-42c8-98f6-68a87cbcf1d9">